### ISSUE TITLE
cudaCheckError() failed : no kernel image is available for execution on the device

### ISSUE THREAD

**Issue by kenanozturk** [ppengtang/pcl.pytorch#8]
Hi,

I am trying to run the code following the instructions. However, I get a cuda check error. Editing the make.sh won't help either.
....
INFO train_net_step.py: 230: 10022 roidb entries
INFO train_net_step.py: 231: Takes 2.30 sec(s) to construct roidb
INFO train_net_step.py: 367: Training starts !
INFO net.py:  52: Changing learning rate 0.000000 -> 0.000167
cudaCheckError() failed : no kernel image is available for execution on the device
...

I installed pytorch with conda and have a TitanV.

Thanks in advance

**Comment by ppengtang** [comment]
I have not met this problem before. But you can refer to [here](https://github.com/roytseng-tw/Detectron.pytorch/issues/168) or try to use Docker.
